_chapters/html.md

```diff
@@ -1,4 +1,221 @@
 ---
 title: HTML
-enabled: false
 ---
+Generally speaking, you should always follow the [W3C HTML5 standard].
+
+## Document Type Declaration
+Always use the [HTML5 doctype], and place it before any other markup (usually
+the `<html>` element):
+
+```html
+<!DOCTYPE html>
+```
+
+The lowercase variation is also valid:
+
+```html
+<!doctype html>
+```
+
+> **Here's why:** The [DTD], or "doctype", tells browsers (and other programs
+> that read HTML) what to expect in your document. In some browsers, the
+> presence or absence of certain DTDs can drastically change the way that your
+> markup, style, and scripts are interpreted. In "[quirks mode]", browsers lack
+> many modern web features and emulate the buggy behavior of sometimes ancient
+> versions.
+
+**Note:** The doctype _must_ be followed immediately (ignoring whitespace) by
+an element such as `<html>`, and not a comment. See [the spec][optional tags]
```

Comment:
I suggest we add a recommendation to add in the "lang" attribute to the `html` tag. If you don't add it, it will fail automate accessibility tests.